src/lib/Cargo/__tests__/__fixtures__/node/Cargo.toml

```diff
@@ -0,0 +1,80 @@
+[package]
```

Comment:
I think we can simplify this fixture. I'd remove the majority of the dependencies and update it to make it clear this is a test fixture

Reply:
done